preprocessing/science/kinematic_models/param2srf.py

```diff
@@ -0,0 +1,204 @@
+import numpy as np
+import os
+import argparse
+
+"""
+Convert param file to srf file
+"""
+
+
+# srf file requires vs and rho, we take the value from PREM file
+PREM_STR = """
+  depth radius   vp    vs    rho     Qu   Qk       P
+   0.0  6371.0   1.45  0.00  1.02    0.0  57823.0    0.0
+   3.0  6368.0   5.80  3.20  2.60  600.0  57823.0    0.0
+  15.0  6356.0   6.80  3.90  2.90  600.0  57823.0    0.3
+  24.4  6346.6   6.80  3.90  2.90  600.0  57823.0    0.6
+  71.0  6300.0   8.08  4.47  3.38  600.0  57823.0    2.2
+  80.0  6291.0   8.08  4.47  3.37  600.0  57823.0    2.5
+ 171.0  6200.0   8.02  4.44  3.36   80.0  57823.0    5.5
+ """
```

Comment:
Running the script currently results in a _ValueError: could not convert string to float: 'depth'_

The reason why this happens is that the file PREM.txt is saved with an additional blank line at the top.
Either change l.26 to `skiprows=2` or maybe better, write l.11 to l.20 as follows:

> PREM_STR = """depth radius   vp    vs    rho     Qu   Qk       P
>    0.0  6371.0   1.45  0.00  1.02    0.0  57823.0    0.0
>    3.0  6368.0   5.80  3.20  2.60  600.0  57823.0    0.0
>   15.0  6356.0   6.80  3.90  2.90  600.0  57823.0    0.3
>   24.4  6346.6   6.80  3.90  2.90  600.0  57823.0    0.6
>   71.0  6300.0   8.08  4.47  3.38  600.0  57823.0    2.2
>   80.0  6291.0   8.08  4.47  3.37  600.0  57823.0    2.5
>  171.0  6200.0   8.02  4.44  3.36   80.0  57823.0    5.5
>  """